<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">



#boldTextIconPushButton {
	border-image: url(:/icons/Letter-B-grey-icon.png);
}
#boldTextIconPushButton:pressed {
	border-image: url(:/icons/Letter-B-pink-icon.png);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="plainTextEdit">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Welcome to this world.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Testing Testing Testing Testing Testing &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="ZoomInPushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/Add-icon.png</normaloff>:/icons/Add-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ZoomOutPushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/Button Delete_256.png</normaloff>:/icons/Button Delete_256.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbFontSize">
        <item>
         <property name="text">
          <string>Very small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Large</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Very large</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbColorSelector">
        <item>
         <property name="text">
          <string>Black</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/pyramid_ball.ico</normaloff>
           <normalon>:/icons/pyramid_ball.ico</normalon>
           <activeoff>:/icons/pyramid_ball.ico</activeoff>
           <activeon>:/icons/pyramid_ball.ico</activeon>
           <selectedoff>:/icons/pyramid_ball.ico</selectedoff>
           <selectedon>:/icons/pyramid_ball.ico</selectedon>:/icons/pyramid_ball.ico</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Red</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/red_light.png</normaloff>
           <normalon>:/icons/red_light.png</normalon>
           <activeoff>:/icons/red_light.png</activeoff>
           <activeon>:/icons/red_light.png</activeon>
           <selectedoff>:/icons/red_light.png</selectedoff>
           <selectedon>:/icons/red_light.png</selectedon>:/icons/red_light.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Green</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/green_light.png</normaloff>
           <normalon>:/icons/green_light.png</normalon>
           <activeoff>:/icons/green_light.png</activeoff>
           <activeon>:/icons/green_light.png</activeon>
           <selectedoff>:/icons/green_light.png</selectedoff>
           <selectedon>:/icons/green_light.png</selectedon>:/icons/green_light.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yellow</string>
         </property>
         <property name="icon">
          <iconset>
           <activeon>:/icons/yellow_light.png</activeon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blue</string>
         </property>
         <property name="icon">
          <iconset resource="../icons.qrc">
           <normaloff>:/icons/ball_blue.png</normaloff>
           <normalon>:/icons/ball_blue.png</normalon>
           <activeoff>:/icons/ball_blue.png</activeoff>
           <activeon>:/icons/ball_blue.png</activeon>
           <selectedoff>:/icons/ball_blue.png</selectedoff>
           <selectedon>:/icons/ball_blue.png</selectedon>:/icons/ball_blue.png</iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boldPushButton">
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boldTextIconPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="newFilePushButton">
        <property name="text">
         <string>New File</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/Add-icon.png</normaloff>:/icons/Add-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadPushButton">
        <property name="text">
         <string>Load</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/folder-blue-icon.png</normaloff>:/icons/folder-blue-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changePasswordPushButton">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/application_x_ms_dos_executable (1).ico</normaloff>:/icons/application_x_ms_dos_executable (1).ico</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="endOfLinePushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/bluePlus.png</normaloff>:/icons/bluePlus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SavePushButton">
        <property name="text">
         <string>Save</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/Downloads-Folder-icon.png</normaloff>:/icons/Downloads-Folder-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SaveClosePushButton">
        <property name="text">
         <string>Save+Close</string>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/Knob-Cancel-icon.png</normaloff>:/icons/Knob-Cancel-icon.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="onlyClosePushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../icons.qrc">
          <normaloff>:/icons/Knob-Cancel-icon.png</normaloff>:/icons/Knob-Cancel-icon.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
